devtools/start-app.bat

```diff
@@ -7,6 +7,7 @@ if !errorlevel! neq 0 (
   exit /B !errorlevel!
 )
 
+set PYTHONPATH=%project_dir%;%PYTHONPATH%
```

Comment:
from PyCharm terminal, `import geoapps` is working because `geoapps` is in the current folder...

Reply:
Thanks Seb, that last one did the trick.  Tested by printing a line in geoh5 init then running devtools/start-app, run first cell of inversion app and getting a print statement (nothing added to environment variables).